### DISCUSSION TITLE
Question regarding soloud flutter feature

### DISCUSSION THREAD

**Discussion by RProduction** [alnitak/flutter_soloud#19] [Q&A]
Hello, it is possible to implement metronome and record audio according to measurement?

Also, how to use fader? Or can I implement multiple bus, parent bus, bus send?

Please I need clarification before I decided to use this library.

**Comment by alnitak** [comment]
Hi,
recording and push audio sampler (to implement metronome) to output device is not supported (yet?).

To use the faders:
`SoLoud().fadeVolume(int handle, double to, double time)`
or 
`SoLoud().fadeGlobalVolume(double to, double time)`
as described better on Soloud docs [here](https://solhsa.com/soloud/faders.html)

`fadeVolume` is used to set a fade to a single sound, while `fadeGlobalVolume` sets the fading to the output. 
By using `fadeVolume` you will notice it after playing the handle, if you set it while playing there are no effect.
So after loading a sound and set a `fadeVolume()` to it, all the subsequent play of that handle, will have the fade effect. 

This behavior is the same for all other faders.

Hope this helps
